spec/unit/plugins/linux/virtualization_spec.rb

```diff
@@ -493,6 +493,25 @@
       expect(@plugin[:virtualization][:systems][:docker]).to eq("guest")
     end
 
+    it "should set docker guest if /proc/self/cgroup exists and there are /system.slice/docker-<hexadecimal> mounts" do
```

Comment:
Would be awesome to have a note about which version of Docker this applies to, either here or in the non-test code.